app/src/main/java/com/gh4a/fragment/SearchFragment.java

```diff
@@ -244,7 +244,7 @@ public void run() {
                 }.start();
             } else {
                 mQuery = cursor.getString(1);
-                mSearch.setQuery(mQuery, false);
+                mSearch.setQuery(mQuery, true);
```

Comment:
I did the current behavior on purpose. Say you want to do a similar search as last time, just slightly modified: my idea for that was that in that case you'd select a previous search entry and modify it. With this change, you'd need to run the search first.

Reply:
I think that doing "a similar search as last time, just slightly modified" is very rare in practice, based also on how I use the search. Probably this is more likely to happen when searching code, so the ability to modify the suggestion before searching could be kept only in that case?

Reply:
Or better, an alternative approach that would make everyone happy could be the one used in Firefox:
![firefox_example](https://user-images.githubusercontent.com/30041551/133970909-722485e0-f874-4a90-aec2-dd39b9cdb48c.jpeg)

Tapping on an entry navigates directly to the website, while clicking on the arrows just copies the URL in the bar, allowing the user to edit it.

Reply:
That sounds like a good idea :+1:

Reply:
Ok, will try to implement it.
To avoid keeping this PR around, I'd remove the second commit so that you can merge only the first, if it's fine for you.

Reply:
I like that idea. I make modifications to my query when searching issues if my first query returned too many (or few) results.